package.json

```diff
@@ -5,22 +5,27 @@
   "private": true,
   "type": "module",
   "sideEffect": false,
-  "packageManager": "pnpm@9.11.0",
+  "packageManager": "pnpm@9.12.1",
   "scripts": {
     "format:write": "prettier --write .",
     "format:check": "prettier --check .",
     "test:all": "vitest",
     "update-deps": "pnpm update -i -r --latest;clear; pnpm licenses ls --json > packages-licenses.json"
   },
   "devDependencies": {
-    "@vitejs/plugin-react": "4.3.1",
-    "@vitest/coverage-v8": "^2.1.1",
-    "@vitest/ui": "2.1.1",
+    "@vitejs/plugin-react": "4.3.2",
+    "@vitest/coverage-v8": "^2.1.3",
+    "@vitest/ui": "2.1.3",
     "prettier": "^3.3.3",
     "prettier-plugin-tailwindcss": "^0.6.8",
     "react-refresh": "^0.14.2",
-    "typescript": "5.6.2",
+    "typescript": "5.6.3",
     "vite-tsconfig-paths": "^5.0.1",
-    "vitest": "2.1.1"
+    "vitest": "2.1.3"
+  },
+  "pnpm": {
+    "overrides": {
+      "cookie": "^0.7.2"
+    }
```

Comment:
I forced `pnpm` to use a specific `cookie` npm pkg version.
Without that, it was using 3 different ones : `0.6.0`, `0.7.1` and `0.7.2`.

it may be due to the fact `pnpm` consider `0.*.*` version different from the classic semver : a minor update is considered "breaking" (so it do not take care of `^` when doing package deps resolution)

But I'm not totatally sure. In the meantime, this should solve our issues (and the forced version is considered non breaking by the pkg manager).